common/lib/capa/capa/templates/clarification.html

```diff
@@ -1,5 +1,6 @@
+<%page expression_filter="h"/>
```

Comment:
This is a new one for me-- I wasn't aware of this feature. Here is the PR associated with it:

https://github.com/edx/edx-platform/pull/6679